source/OptimaJet.DWKit.StarterApplication/Repositories/OrganizationInviteRequestRepository.cs

```diff
@@ -10,16 +10,17 @@
 
 namespace OptimaJet.DWKit.StarterApplication.Repositories
 {
-    public class OrganizationInviteRequestRepository : ControllerRepository<OrganizationInviteRequest>
+    public class OrganizationInviteRequestRepository : BaseRepository<OrganizationInviteRequest>
     {
         private readonly IBackgroundJobClient backgroundJobClient;
 
         public OrganizationInviteRequestRepository(
             ILoggerFactory loggerFactory,
             IJsonApiContext jsonApiContext,
             IDbContextResolver contextResolver,
+            CurrentUserRepository currentUserRepository,
```

Comment:
the currentUserRepository was added in a prior PR to avoid a circular dependency between Repositories and Services.

This CurrentUserRepository only depends on the CurrentUserContext (and the jsonapi-provided stuff).